pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -132,9 +132,18 @@ func GetAvailableNotifiers() []*NotifierPlugin {
 						},
 					},
 				},
+				{ // New in 9.3.
+					Label:        "Title",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Description:  "Templated title of the message",
+					Placeholder:  channels.DefaultMessageTitleEmbed,
+					PropertyName: "title",
+				},
 				{ // New in 8.0.
 					Label:        "Message",
 					Element:      ElementTypeTextArea,
+					Description:  "Templated description of the message",
```

Comment:
This field is pretty self-explanatory. In any case, I think it's better to say something like:

```suggestion
					Description:  "Custom DingDing message. You can use template variables.",
```